### PR TITLE
feat: replace CRA by Vite + React

### PR DESCRIPTION
**Pull request by d-klotz**
This PR aims to replace CRA by Vite.

### CODE REVIEW

File: packages/cfa-template/template/backend/src/configs/dev.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "MONGO_URI": "mongodb://root:rootpassword@localhost:27017?retryWrites=true&w=majority",
     "REDIRECT_URI": "http://localhost:3000/redirect",
 
-    "HUBSPOT_CLIENT_ID": "",
-    "HUBSPOT_CLIENT_SECRET": "",
-    "HUBSPOT_SCOPE": "crm.objects.contacts.read crm.objects.companies.read crm.objects.companies.write crm.objects.contacts.write crm.schemas.deals.read crm.objects.deals.read",
+    "HUBSPOT_CLIENT_ID": "37864c83-856c-42b1-95ad-8a5f681fc8d1",
```
Comment (seanspeaks) [inline packages/cfa-template/template/backend/src/configs/dev.json:7]: I think we should omit this (gitignore didn't ignore 😅)

File: packages/cfa-template/template/frontend/.eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "@friggframework/eslint-config"
+//    "extends": "@friggframework/eslint-config"
```
Comment (seanspeaks) [inline packages/cfa-template/template/frontend/.eslintrc.json:2]: Reason for this?

File: packages/cfa-template/template/frontend/src/components/Integration/ModalFormBasedAuth.js [hunk 3]
```diff
@@ -2,184 +2,207 @@ import Form from '@rjsf/core';
...
+                  <path
+                    className="opacity-75"
+                    fill="currentColor"
+                    d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
+                  ></path>
+                </svg>
+              </div>
+            ) : null}
+
+            <Form
```
Comment (seanspeaks) [inline packages/cfa-template/template/frontend/src/components/Integration/ModalFormBasedAuth.js:181]: Thought we had swapped out already (for JSONforms). Different place?

**Review by seanspeaks — CHANGES_REQUESTED** [review]
Throwing some comments in for now, need to review on comp later

File: packages/cfa-template/template/frontend/.eslintrc.json (hunk 2 above)
Comment (d-klotz) [reply packages/cfa-template/template/frontend/.eslintrc.json:2]: Still did not check why but it's throwing an error during build

File: packages/cfa-template/template/frontend/src/components/Integration/ModalFormBasedAuth.js (hunk 3 above)
Comment (d-klotz) [reply packages/cfa-template/template/frontend/src/components/Integration/ModalFormBasedAuth.js:181]: Yes, we have a JSONForms version of this, I'll remove this one.

File: packages/cfa-template/template/backend/src/configs/dev.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "AES_KEY_ID": "",
+    "MONGO_URI": "mongodb://root:rootpassword@localhost:27017?retryWrites=true&w=majority",
+    "REDIRECT_URI": "http://localhost:3000/redirect",
+
+    "HUBSPOT_CLIENT_ID": "",
+    "HUBSPOT_CLIENT_SECRET": "",
+    "HUBSPOT_SCOPE": "oauth timeline crm.objects.contacts.read crm.objects.contacts.write",
+
+    "SALESFORCE_CONSUMER_KEY": "",
+    "SALESFORCE_CONSUMER_SECRET": ""
```
Comment (igorschechtel) [inline packages/cfa-template/template/backend/src/configs/dev.json:12]: Should this be included?

File: packages/cfa-template/template/frontend/package.json [hunk 5]
```diff
@@ -31,6 +24,7 @@
     ]
   },
   "dependencies": {
```
Comment (igorschechtel) [inline packages/cfa-template/template/frontend/package.json:26]: Do we need all these dependencies?

**Review by igorschechtel — APPROVED** [review]

File: packages/cfa-template/template/backend/src/configs/dev.json (hunk 4 above)
Comment (seanspeaks) [reply packages/cfa-template/template/backend/src/configs/dev.json:12]: We could remove 👌

**Review by seanspeaks — APPROVED** [review]
I ran no tests to confirm it builds and runs, but assuming that was smoke screen tested I'm good to go 👌

We can clean up further later

File: packages/cfa-template/template/frontend/package.json (hunk 5 above)
Comment (d-klotz) [reply packages/cfa-template/template/frontend/package.json:26]: For now yes, but we will be removing as we go.

File: packages/cfa-template/template/backend/src/configs/dev.json (hunk 4 above)
Comment (d-klotz) [reply packages/cfa-template/template/backend/src/configs/dev.json]: removed

**Pull request merged**